web_copy_confirm/static/src/js/web_copy_confirm.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018 DynApps <http://www.dynapps.be>
+// @author Stefan Rijnhart <stefan@opener.amsterdam>
+// @author Robin Conjour <rconjour@demolium.com>
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define("web_copy_confirm.web_copy_confirm", function (require) {
+    "use strict";
+
+    const Core = require("web.core");
+    const FormController = require("web.FormController");
+    const Dialog = require("web.Dialog");
+    const _t = Core._t;
+
+    return FormController.include({
+        _onDuplicateRecord: async function (values, confirmed = false) {
```

Comment:
I'm a little worried about adding the additional parameter. If another module overrides this method, that could lead to problems because `confirmed` is never passed as true. Is there another way, for instance by letting the button call a new entry point (for instance, _onDuplicateRecordConfirm)?

Reply:
Agreed, overrided the function in the action list.